packages/ckeditor5-table/src/converters/table-cell-refresh-post-fixer.js

```diff
@@ -25,79 +25,89 @@ export default function injectTableCellRefreshPostFixer( model ) {
 function tableCellRefreshPostFixer( model ) {
 	const differ = model.document.differ;
 
-	// Stores cells to be refreshed so the table cell will be refreshed once for multiple changes.
-	const cellsToRefresh = new Set();
-
-	// Counting the paragraph inserts to verify if it increased to more than one paragraph in the current differ.
-	let insertCount = 0;
+	const changesForCells = new Map();
+	const changes = [ ...differ.getChanges() ];
```

Comment:
The `getChanges()` returns a fresh copy of array so there is no need for copying it again.

Reply:
Yep, debugging lef-tovleftoverer.